hangmanweb.js

```diff
@@ -0,0 +1,76 @@
+document.addEventListener("DOMContentLoaded", () => {
```

Comment:
Overall impressive app with good functionality and without major glitches. I like the theme and also readable code that you wrote for js. One caveat that I found is lack os mechanism to prevent user from inputting same letter after wrong guesses but it's minor. Keep up the good work and I'm looking forward to see future projects!